src/calculateRentalCost.js

```diff
@@ -1,10 +1,22 @@
+const carCosts = 40;
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return carCosts * days - 50;
+  }
+
+  if (days >= 3) {
+    return carCosts * days - 20;
+  }
+
+  if (days < 3 && days > 0) {
+    return carCosts * days;
```

Comment:
The check for `days > 0` is redundant since the function should not be called with negative numbers. If you want to ensure that `days` is a positive number, it's better to handle that at the beginning of the function with a guard clause.